src/osd/scrubber/pg_scrubber.cc

```diff
@@ -1410,6 +1412,8 @@ void PgScrubber::scrub_compare_maps()
 
     dout(2) << __func__ << ": primary (" << m_pg->get_primary() << ") has "
 	    << m_primary_scrubmap.objects.size() << " items" << dendl;
+    primary_objects = m_primary_scrubmap.objects.size();
```

Comment:
I would just skip 'primary_objects' altogether (the only good reason to have it would
have been to use it in the dout() in the previous line)